extras/bindings/c/src/SolverInterfaceC.cpp

```diff
@@ -189,6 +189,12 @@ int precicec_getDataID(const char *dataName, int meshID)
   return impl->getDataID(stringDataName, meshID);
 }
 
+int precicec_isMeshConnectivityRequired(int meshID)
+{
+  PRECICE_CHECK(impl != nullptr, errormsg);
+  return impl->isMeshConnectivityRequired(meshID);
```

Comment:
```suggestion
  if (impl->isMeshConnectivityRequired(meshID)) {
    return 1;
  }
  return 0;  
```
At least, this is how we do it for similar API functions.